structuredefinitions/UKCore-Specimen.xml

```diff
@@ -81,21 +81,17 @@
         <profile value="http://hl7.org/fhir/StructureDefinition/specimen-specialHandling" />
       </type>
     </element>
-    <element id="Specimen.collection.collector.extension:collectionCollectorR5">
-      <path value="Specimen.collection.collector.extension" />
-      <sliceName value="collectionCollectorR5" />
-      <short value="The person who collected the specimen. This is an R5 backport." />
-      <definition value="Person who collected the specimen. This is an R5 backport" />
+    <element id="Specimen.collection.extension:CollectorR5">
+      <path value="Specimen.collection.extension" />
+      <sliceName value="CollectorR5" />
```

Comment:
slice name to start with lower case. collectorR5

Reply:
fixed